jekyll/_cci2/vs-code-extension-quickstart.adoc

```diff
@@ -0,0 +1,117 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+---
+= Get started with the VS Code extension
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A quickstart tutorial for the CircleCI VS Code extension.
+:icons: font
+:toc: macro
+:toc-title:
+
+This tutorial is a quickstart guide to setting up and using the official CircleCI VS Code extension. You can read more about the extension's capabilities in the xref:vs-code-extension-overview#[overview]. 
+
+[#prerequisits]
+== Prerequisites
+
+Before starting this tutorial, make sure you have:
+
+* Installed the Visual Studio Code editor
++
+NOTE: The CircleCI extension is compatible with VS Code version `1.67` or above.
+
+* A CircleCI account (you can link:https://circleci.com/signup/[sign up for free])
+
+* A xref:create-project#[project] you follow that is building on CircleCI
+
+* CircleCI server users: your self-hosted server URL
+
+[#install-the-extension]
+== 1. Install the extension
+
+There are two ways to install the CircleCI extension:
+
+* In VS Code, click the **Extensions** icon on the left-hand Activity Bar, or open **View > Extensions**.
+
+* Or, download the extension from the link:https://marketplace.visualstudio.com/items?itemName=circleci.circleci[VS Code Marketplace].
+
+[#authenticate-into-circleci]
+== 2. Authenticate into CircleCI
+
+. In VS Code, open the CircleCI panel by clicking on the CircleCI icon on the activity bar, then click “Log in”.
+
+. You will be prompted to enter a xref:managing-api-tokens#overview[personal API token]. You may create one in your CircleCI link:https://app.circleci.com/settings/user/tokens[**User Settings > Personal API Tokens**.]
++
+**CircleCI server**: Make sure to check the "I'm a CircleCI Server customer," then enter your self-hosted server URL.
++
+image::{{site.baseurl}}/assets/img/docs/vs_code_extension_login.png[Log in to CircleCI from VS Code]
+
+[#set-up-your-project-in-vs-code]
+== 3. Set up your project in VS Code
+
+NOTE: The pipelines manager does not yet support GitLab projects.
+
+If the root of your workspace is a CircleCI project or contains a project that you follow, your project will be detected automatically, and the pipelines panel will be populated with your most recent pipelines.
+
+Otherwise, you may manually select a project by opening the extension's settings either through the VS Code command `CircleCI: Open Settings`, or by clicking on the settings icon (⚙️) at the top of the pipelines panel.
+
+NOTE: Support for link:https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspaces] will be available soon.
```

Comment:
@BeFunes Do you think we should still mention multi-root workspace support specifically? My guess is it's already implied by "contains one or more CircleCI projects," but wasn't sure if readers might be looking out for that term or if we want it in there anyway (for search / SEO purposes)?

Reply:
Correct, when we say "one or more CircleCI projects" that implies it. 
I don't think it's necessary to mention multi-root workspaces, just because it's not this flashy feature. VS Code users have come to expect it from extensions in general, we just were a bit behind until now. 
But if you think it'd help SEO, no objections from me!

Reply:
I just realised I put my initial comment in the wrong place. 

To clarify:
* The `Note` needs to be deleted completely.
    ```
    NOTE: Support for link:https://code.visualstudio.com/docs/editor/multi-root-workspaces[multi-root workspaces] will be available soon.
    ```

* The initial two sentences must be replaced with the new content
**Before**: 
![Screenshot 2023-06-21 at 09 13 08](https://github.com/circleci/circleci-docs/assets/19594309/e4882d45-659e-48e2-9f11-97a6ccc27f87)
**After**: 
    ```
    If your VS Code workspace contains one or more CircleCI projects, the extension will detect them automatically, and the Pipelines Panel will be populated with your most recent pipelines. If no project is detected, open the settings page (through the VS Code command `CircleCI: Open Settings`, or by clicking on the settings icon (⚙️) at the top of the Pipelines Panel), and select your projects manually. Please note that your manual selection will only be persisted if no projects are detected automatically.
    ```